documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -214,11 +216,12 @@ spec:
 EOF
 ----
 <1> Allowed values are `ReadWriteOnce` and `ReadWriteMany`.
-<2> Specify the VMware data storage MOR, the {rhv-short} storage domain UUID, or the {osp} `volume_type` UUID. For example, `f2737930-b567-451a-9ceb-2887f6207009`.
+<2> Specify the VMware data storage MOR, the {rhv-short} storage domain UUID, or the {osp} `volume_type` UUID. For example, `f2737930-b567-451a-9ceb-2887f6207009`. For OVA, use `name: Dummy storage for source provider provider-ova` instead of `id: <source_datastore>`.
```

Comment:
how about dropping the `For example, f2737930-b567-451a-9ceb-2887f6207009` part? we don't do that elsewhere..